[ko] translate 'migrating-from-dockershim/index' (#36865)
kubernetes/website: content/ko/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

---
title: "도커심으로부터 마이그레이션"
weight: 10
content_type: task
no_list: true
---

<!-- overview -->

이 섹션은 도커심에서 다른 컨테이너 런타임으로

> 이주할 때에 알아야할 정보를 제공한다.
> 
> [도커심 폐지](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation) 발표 이후
> 쿠버네티스 1.20에서 이것이 다양한 워크로드와 쿠버네티스 설치에
> 어떤 영향을 미칠지에 대한 질문이 있었다. [도커심 제거 FAQ](/blog/2022/02/17/dockershim-faq/)는

Reviewer: ```suggestion
마이그레이션할 때 알아야 할 정보를 제공한다.

[도커심 사용 중단](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation) 발표 이후
쿠버네티스 1.20에서 이것이 다양한 워크로드와 쿠버네티스 설치에
어떤 영향을 미칠지에 대한 질문이 있었다. [도커심 제거 FAQ](/blog/2022/02/17/dockershim-faq/)는
```
용어 표기를 통일합니다.

문제를 더 잘 이해하는 데 도움이 된다.

도커심은 쿠버네티스 릴리즈 v1.24부터 제거되었다.
컨테이너 런타임으로 도커심을 통해 도커 엔진을 사용하는 경우, v1.24로
업그레이드하려면 다른 런타임으로 마이그레이션하거나 다른 방법을 찾아 도커 엔진 지원을 받는 것이 좋다.
옵션을 알고싶다면 [컨테이너 런타임](/docs/setup/production-environment/container-runtimes/) 섹션을 확인한다.
마이그레이션 중 문제를 마주한다면
[문제를 보고](https://github.com/kubernetes/kubernetes/issues)해야한다. 따라서 문제를 시기적절하게
해결할 수 있으며, 클러스터는 도킹 스테이션 제거에
대비할 수 있다.

클러스터에 두 가지 이상의 노드가 있을 수 있지만
이는 일반적인 구성은 아니다.

다음 작업을 통해 마이그레이션을 수행할 수 있다:

- [도커심 제거가 당신에게 영향을 미치는지 확인하기](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
- [Docker Engine 노드를 도커심에서 cri-dockerd로 마이그레이션하기](/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)
- [도커심으로부터 보안 에이전트와 원격 측정을 마이그레이션하기](/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents/)

## {{% heading "whatsnext" %}}

> * 컨테이너 런타임에 대한 옵션을 이해하기 위해 [컨테이너 런타임](/docs/setup/production-environment/container-runtimes/)을 확인한다.
> * 도커심의 사용 중지 및 제거에 대한 논의를 추적하는 [깃허브 이슈](https://github.com/kubernetes/kubernetes/issues/106917)가 있디.
> * 도커심에서 마이그레이션하는 것에 관련한 결함이나 다른 기술적 문제를 발견한다면, 쿠버네티스 프로젝트에 [이슈를 남길](https://github.com/kubernetes/kubernetes/issues/new/choose) 수 있다.

Reviewer: ```suggestion
* 컨테이너 런타임에 대한 옵션을 이해하기 위해
[컨테이너 런타임](/docs/setup/production-environment/container-runtimes/)을 확인한다.
* 도커심의
사용 중단 및 제거에 대한 논의를 추적하는
[깃허브 이슈](https://github.com/kubernetes/kubernetes/issues/106917)가 있다.
* 도커심에서 마이그레이션하는 것에 관한
결함이나 다른 기술적 문제를 발견한다면,
쿠버네티스 프로젝트에 [이슈를 남길 수 있다.](https://github.com/kubernetes/kubernetes/issues/new/choose)
```

원문 형태 유지를 위한 개행과 사소한 용어 변경입니다.

---
title: "도커심으로부터 마이그레이션"
weight: 10
content_type: task
no_list: true
---

<!-- overview -->

이 섹션은 도커심에서 다른 컨테이너 런타임으로
마이그레이션할 때에 알아야 할 정보를 제공한다.

[도커심 사용 중단](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation) 발표 이후
쿠버네티스 1.20에서 이것이 다양한 워크로드와 쿠버네티스 설치에
어떤 영향을 미칠지에 대한 질문이 있었다. [도커심 제거 FAQ](/blog/2022/02/17/dockershim-faq/)는
문제를 더 잘 이해하는 데 도움이 된다.

도커심은 쿠버네티스 릴리즈 v1.24부터 제거되었다.

> 컨테이너 런타임으로 도커심을 통해 도커 엔진을 사용하는 경우, v1.24로
> 업그레이드하려면 다른 런타임으로 마이그레이션하거나 다른 방법을 찾아 도커 엔진 지원을 받는 것이 좋다.

Reviewer 2: ```suggestion
컨테이너 런타임으로 도커 엔진을 통한 도커심을 사용하는 상황에서 v1.24로
업그레이드하려는 경우, 다른 런타임으로 마이그레이션하거나 다른 방법을 찾아 도커 엔진 지원을 받는 것이 좋다.
```
자연스럽게 수정해보았습니다.

> 옵션을 알고싶다면 [컨테이너 런타임](/docs/setup/production-environment/container-runtimes/) 섹션을 확인한다.

Reviewer 2: ```suggestion
선택 가능한 옵션은0 [컨테이너 런타임](/ko/docs/setup/production-environment/container-runtimes/) 섹션에서 확인한다.
```
한글 문서가 있으므로, 한글 문서로 링크를 변경해주세요~

https://kubernetes.io/ko/docs/setup/production-environment/container-runtimes/

> 마이그레이션 중 문제를 마주한다면
> [문제를 보고](https://github.com/kubernetes/kubernetes/issues)해야한다. 따라서 문제를 시기적절하게

Reviewer 2: ```suggestion
마이그레이션 중 문제를 마주한다면
[문제를 보고](https://github.com/kubernetes/kubernetes/issues)하면 좋다. 이를 통해 문제를 시기적절하게
```
의무 사항처럼 느껴질 수 있어서, 약화시켜보았습니다.

> 해결할 수 있으며, 클러스터는 도킹 스테이션 제거에
> 대비할 수 있다.

Reviewer 2: ```suggestion
해결할 수 있으며, 클러스터도 도커심 제거에
대비할 수 있다.
```
아마 기계번역에 의해서 포함된 문구인 듯 합니다. 리뷰 과정을 쉽게 하기 위해서, 기여자가 PR 올리기 전에 자체적으로 내용을 읽어보고 오류는 없는지 내용이 자연스러운지 살펴보는 노력도 필요합니다. :)

> 클러스터에 두 가지 이상의 노드가 있을 수 있지만
> 이는 일반적인 구성은 아니다.

Reviewer 2: ```suggestion
클러스터는 두 종류 이상의 노드들을 포함할 수 있지만
이는 일반적인 구성은 아니다.
```

아마 원문의 내용은 노드별로 다른 컨테이너 런타임을 사용하는 경우를 의미하는 것 같습니다. 

직역을 하니 단순히 노드 개수가 여러 개인 상황을 의미하는 것 같이 느껴지기도 하네요.
의미를 살리기 위해서 조금 수정해보았습니다. :)

Your cluster might have more than one kind of node, although this is not a common configuration.

> 다음 작업을 통해 마이그레이션을 수행할 수 있다:

Reviewer 2: ```suggestion
다음 작업을 통해 마이그레이션을 수행할 수 있다.
```
한글 문장의 구성상 : 로 끝나는 경우가 없다고 하여,
가급적 . 등으로 적절하게 수정하고 있습니다.

- [도커심 제거가 당신에게 영향을 미치는지 확인하기](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)

> - [Docker Engine 노드를 도커심에서 cri-dockerd로 마이그레이션하기](/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)

Reviewer 2: ```suggestion
- [도커 엔진 노드를 도커심에서 cri-dockerd로 마이그레이션하기](/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)
```
문서 내 일관성을 위한 제안입니다.

> - [도커심으로부터 보안 에이전트와 원격 측정을 마이그레이션하기](/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents/)

Reviewer 2: ```suggestion
- [도커심으로부터 보안 및 텔레메트리 에이전트 마이그레이션하기](/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents/)
```
현재 문구에서는 조금 더 명확한 표현을 위해 
telemetry -> 텔레메트리 로 표현하면 어떨까 싶습니다. 

측정 자체에 대한 마이그레이션이 아니라, 측정을 위한 에이전트(Datadog 등등)를 마이그레이션하는 것을 의미하기 때문입니다.

Migrating telemetry and security agents from dockershim

Author: 특히 이부분 원문과 비교해보면서 어떻게 변역할지 잘 모르겠었는데 요렇게 해야하군요..! 감사합니다

## {{% heading "whatsnext" %}}

* 컨테이너 런타임에 대한 옵션을 이해하기 위해

> [컨테이너 런타임](/docs/setup/production-environment/container-runtimes/)을 확인한다.

Reviewer 2: ```suggestion
  [컨테이너 런타임](/ko/docs/setup/production-environment/container-runtimes/)을 확인한다.
```

> * 도커심의
> 사용 중단 및 제거에 대한 논의를 추적하는
> [깃허브 이슈](https://github.com/kubernetes/kubernetes/issues/106917)가 있다.
> * 도커심에서 마이그레이션하는 것에 관한
> 결함이나 다른 기술적 문제를 발견한다면,
> 쿠버네티스 프로젝트에 [이슈를 남길 수 있다.](https://github.com/kubernetes/kubernetes/issues/new/choose)

Reviewer 2: ```suggestion
* 도커심의
  사용 중단 및 제거에 대한 논의를 추적하는
  [깃허브 이슈](https://github.com/kubernetes/kubernetes/issues/106917)가 있다.
* 도커심에서 마이그레이션하는 것에 관한
  결함이나 다른 기술적 문제를 발견한다면,
  쿠버네티스 프로젝트에 [이슈를 남길 수 있다.](https://github.com/kubernetes/kubernetes/issues/new/choose)
```

원문의 구성을 유지할 필요가 있습니다. :)
